tools/data_builder.py

```diff
@@ -57,9 +57,16 @@ def title_to_name_date(line):
     if date_text in ("Date Unknown", "Unknown Date"):
         return name, "", "Unknown Date"
 
-    date_found = date_regex.search(date_text).group()
+    date_search = date_regex.search(date_text)
+    date_found = date_text.lower().replace("(believed to be)", "").strip()
+    if date_search:
+        date_found = date_search.group()
+
     date = parse(date_found).strftime("%Y-%m-%d")
```

Comment:
I would do something like `if '202' not in date_found: date_found += ' 2020'`.

Reply:
Yeah I tried that and it didn't really work but I'll tinker again because I royally fucked some of this up 🤦‍♀️

Reply:
Okay I think I unfucked it